governator-core/src/main/java/com/netflix/governator/ModuleBuilder.java

```diff
@@ -0,0 +1,238 @@
+     */
+    public static ModuleBuilder fromModule(Module module) {
+        return new ModuleBuilder(module);
+    }
+    
+    public static ModuleBuilder fromModules(Module firstModule, Module secondModule, Module ... additionalModules) {
+        List<Module> modules = new ArrayList<>();
+        modules.add(firstModule);
+        modules.add(secondModule);
+        if (null != additionalModules) {
+            modules.addAll(Arrays.asList(additionalModules));
+        }
+        return new ModuleBuilder(Modules.combine(modules));
+    }
+    
+    public static ModuleBuilder fromModules(List<Module> modules) {
+        return new ModuleBuilder(Modules.combine(modules));
+    }
+    
+    public static ModuleBuilder createDefault() {
+        return fromModule(new Module() {
+            @Override
+            public void configure(Binder binder) {
+            }
+        });
+    }
+    
+    private ModuleBuilder(Module module) {
+        this.module = module;
+    }
+    
+    /**
+     * Override all existing bindings with bindings in the provided modules.
+     * This method uses Guice's build in {@link Modules#override} and is preferable
+     * to using {@link Modules#override}.  The approach here is to attempt to promote 
+     * the use of {@link Modules#override} as a single top level override.  Using
+     * {@link Modules#override} inside Guice modules can result in duplicate bindings 
+     * when the same module is installed in multiple placed. 
+     * @param modules
+     */
+    public ModuleBuilder overrideWith(Module ... modules) {
+        return overrideWith(Arrays.asList(modules));
+    }
+    
+    /**
+     * @see ModuleBuilder#overrideWith(Module...)
+     */
+    public ModuleBuilder overrideWith(Collection<Module> modules) {
+        this.module = Modules.override(module).with(modules);
+        return this;
+    }
+    
+    /**
+     * Add additional bindings to the module tracked by the DSL
+     * @param modules
+     */
+    public ModuleBuilder combineWith(Module ... modules) {
+        List<Module> m = new ArrayList<>();
+        m.add(module);
+        m.addAll(Arrays.asList(modules));
+        this.module = Modules.combine(m);
+        return this;
+    }
+    
+    /**
+     * Iterator through all elements and invoke a visitor.  This method
+     * is mean for side-effect free operations such as logging the current 
+     * state of bindings.
+     * @param visitor
+     */
+    public ModuleBuilder forEachElement(ElementVisitor<Void> visitor) {
+        for (Element element : Elements.getElements(module)) {
+            element.acceptVisitor(visitor);
+        }
+        return this;
+    }
+    
+    /**
+     * Log a warning that static injection is being used.  Static injection is considered a 'hack'
+     * to alllow for backwards compatibility with non DI'd static code.
+     */
+    public ModuleBuilder warnOfStaticInjections() {
+        return forEachElement(new WarnOfStaticInjectionVisitor());
```

Comment:
Hah, you'd already done this.  Personally, I'd rather not have these small wrappers and instead have people use forEachElement directly.